examples/lbfgs/driver.py

```diff
@@ -73,27 +70,29 @@ def set_weights(new_weights):
   def net_reinitialization(net_vars):
     return net_vars
 
-  # Create a reusable variable for the network.
+  # Register the network with Ray and create a reusable variable for it.
+  ray.register_class(LinearModel)
   ray.reusables.net_vars = ray.Reusable(net_initialization, net_reinitialization)
 
   # Load the weights into the network.
   def load_weights(theta):
-    sess, _, _, _, _, get_weights, set_weights = ray.reusables.net_vars
-    set_weights([theta[:w_size].reshape(w_shape), theta[w_size:].reshape(b_shape)])
+    net = ray.reusables.net_vars
```

Comment:
Please get rid of load_weights, that is the whole point of having TensorFlowVariables